package.json

```diff
@@ -170,7 +170,7 @@
     "firefox": "npm run bundle:webpackAddon && jpm run --prefs ./dev-prefs.json",
     "test": "npm-run-all test:*",
     "pretest": "npm run bundle && npm run copyTestImages && npm run copyTopSitesJson",
-    "test:lint": "eslint --ext=.js,.jsx,.json . && sass-lint -v -q",
+    "test:lint": "ls -las ./node_modules/.bin && eslint --ext=.js,.jsx,.json . && sass-lint -v -q",
```

Comment:
I managed to crawl through Travis-CI and killed the build cache for this PR which seemingly "fixed" the ESLint not found errors. Feel free to remove this pesky `ls -las` debug hack.